src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -133,13 +133,10 @@ const getAuthenticatorData = function (authenticator, isVerifyAuthenticator) {
 
   case AUTHENTICATOR_KEY.IDP: {
     const idpName =  authenticator.relatesTo?.displayName;
-    const factorDescription = idpName
-      ? loc('oie.idp.authenticator.description', 'login', [idpName])
-      : loc('oie.idp.authenticator.description.default', 'login');
     Object.assign(authenticatorData, {
       description: isVerifyAuthenticator
         ? ''
-        : factorDescription,
+        : loc('oie.idp.authenticator.description', 'login', [idpName]),
```

Comment:
is the other string getting deleted ? `oie.idp.authenticator.description.default`..

Reply:
You commented too quickly! 😄 

This string [never existed](https://github.com/okta/okta-signin-widget/pull/1707/files#r585108406). We missed it during code review, but _luckily_ we would never encounter the case where the `*.default` string would be used.